snow/engine/snowman/syncer/state_syncer.go

```diff
@@ -248,12 +253,21 @@ func (ss *stateSyncer) AcceptedStateSummary(ctx context.Context, nodeID ids.Node
 		newWeight, err := math.Add64(weight, ws.weight)
 		if err != nil {
 			ss.Ctx.Log.Error("failed to calculate the Accepted votes",
+				zap.Stringer("summaryID", summaryID),
```

Comment:
since we're cleaning up these logs, should the message be something like "failed to increment summary weight"?